### DISCUSSION TITLE
Failing TestHarness test - libquadmath

### DISCUSSION THREAD

**Discussion by cyshi17** [idaholab/moose#16800] [Q&A Getting Started] [answered]
Hello,

I'm running into an error with make -j 8, following the instructions for troubleshooting. I've tried updating and reinstalling moose, and the "hello world" example in the build troubleshooting runs fine. I've included a screenshot of the error I'm getting. 
![image](https://user-images.githubusercontent.com/77810388/105612912-20faa080-5d8d-11eb-81bd-06acd716f97e.png)
Any help would be greatly appreciated. Thank you!

**Comment by GiudGiud** [comment]
Hello

It looks like you're missing libquadmath.
Please install it with "conda install libquadmath" or with your preferred package manager. 

Best,
Guillaume

**Reply by cyshi17** [reply]
Hi Guillaume,

Thank you for your suggestion! Unfortunately I can't seem to install it, as I get the error below. I've looked on anaconda.org as the printout suggests but it does not seem to exist in the cloud. I am using Conda to install this.
![image](https://user-images.githubusercontent.com/77810388/105613432-76847c80-5d90-11eb-85f0-5b15a8499fdb.png)

**Comment by GiudGiud** [comment, marked as answer]
sudo apt install libquadmath0 should work

**Reply by GiudGiud** [reply]
not sure why I thought conda had it

**Reply by cyshi17** [reply]
It worked! Thank you so much

**Comment by Clark-domain** [comment]
I came across the same problem, and did it like this in the root account of Fedora Linux:

sudo yum install libquadmath      #note: not using libquadmath0

It worked.
